tuf/log.py

```diff
@@ -104,18 +105,25 @@
 console_handler = None
 
 # Set the built-in file handler.  Messages will be logged to
-# '_DEFAULT_LOG_FILENAME', and only those messages with a log level of
+# 'tuf.conf.LOG_FILENAME', and only those messages with a log level of
 # '_DEFAULT_LOG_LEVEL'.  The log level of messages handled by 'file_handler'
-# may be modified with 'set_filehandler_log_level()'.  '_DEFAULT_LOG_FILENAME'
+# may be modified with 'set_filehandler_log_level()'.  'tuf.conf.LOG_FILENAME'
 # will be opened in append mode.
-file_handler = logging.FileHandler(_DEFAULT_LOG_FILENAME)
-file_handler.setLevel(_DEFAULT_FILE_LOG_LEVEL)
-file_handler.setFormatter(formatter)
+if tuf.conf.ENABLE_FILE_LOGGING:
+  file_handler = logging.FileHandler(tuf.conf.LOG_FILENAME)
+  file_handler.setLevel(_DEFAULT_FILE_LOG_LEVEL)
+  file_handler.setFormatter(formatter)
+else:
+  pass
+
 
 # Set the logger and its settings.
 logger = logging.getLogger('tuf')
 logger.setLevel(_DEFAULT_LOG_LEVEL)
-logger.addHandler(file_handler)
```

Comment:
I'm wondering if this, and the previous if can be merged together for simplicity. They check for the same condition.